modules/apps/frontend-taglib/frontend-taglib-clay-sample-web/src/main/resources/META-INF/resources/js/ClaySampleToastAlert.js

```diff
@@ -54,6 +64,14 @@ export default function ClaySampleToastAlert() {
 					>
 						{Liferay.Language.get('fail-submit')}
 					</ClayButton>
+
+					<ClayButton
+						_dismissible="false"
```

Comment:
setting `_dismissible` parameter probably is not related to the test. No need to introduce unnecessary parameters for the test...unless you were planning to use this for another test?